velox/docs/develop/unsaferow.rst

```diff
@@ -20,17 +20,18 @@ The 'null bits' section contains one bit per column. 0 bit indicates a
 non-null value. 1 bit indicates a null value.
 
 The 'fixed-width data' section contains 8 bytes per column. Values of the
-fixed-width columns (booleans, integers, floating point numbers) are stored
-directly. These values must fit within 8 bytes. Long decimal columns are not
-supported.
+fixed-width columns (booleans, integers, floating point numbers, short decimals) are stored
```

Comment:
A reader might be confused why long decimals are not considered fixed-width, because the actually are fixed-width. I suggest to clarify this as follows.

```
Values of fixed-width columns that fit within 8 bytes (booleans, integers,...) are stored directly.

Values of the variable-width columns (strings, arrays, maps) and fixed-width columns that are wider than 8 bytes (long decimals) are split between...
```